tests/functional/disaster-recovery/regional-dr/test_deploy_rdr.py

```
@@ -0,0 +1,454 @@
+            )
+            log.info(f"Switching to cluster context (index: {mce_cluster_index})")
+            config.switch_ctx(mce_cluster_index)
+
+            try:
+                deployment.do_deploy_mce()
+                log.info(
+                    f"✓ MCE deployment completed successfully on {mce_cluster_name}"
+                )
+                mce_deployment_results.append((mce_cluster_name, True, None))
+            except Exception as e:
+                error_msg = f"MCE deployment failed on {mce_cluster_name}: {str(e)}"
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)